src/Pages/AboutUs/AboutUs.css

```
@@ -0,0 +1,222 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,700;1,300&&display=swap');
+
+section {
+    padding-left: 100px;
+    padding-right: 100px;
+    padding-top: 20px;
+}
+
+.officer-body h1{ 
+    font-family: 'Montserrat', sans-serif;
+    font-weight: 700;
+    color:#124559;
+    font-size: 40px;
+}
+
+.officer-body h2, h3, p {
+    min-width: 200;
+}
+
+.card{
+    max-width: 250px;
+    max-height: 325px;
+    min-height: 325px;
+    display: block;
+    background-color: #ff930a;
+    border-radius: 9px;
+    margin-bottom: 20px;
+    box-shadow: 0px 0px 10px 2px grey;
+    color: whitesmoke;
+    font-family: 'Montserrat', sans-serif;  
+    border: transparent;
+}
+
+.top-card{
+    background-position: center;
+    background-size: cover;
+    max-width: 250px;
+    max-height: 250px;
+    min-height: 250px;
+    opacity:100%;
+    border-radius: 9px 9px 0px 0px;
+}
+
+.bottom-card{
+    max-width: 250px;
+    max-height: 250px;
+    min-height: 250px;
+    background-color: #0f80aa;
+    color: #f3f3f3;
+    opacity: 0%;
+    border-radius: 9px 9px 0px 0px;
+    padding: 10px 10px 0px 10px;
+}
+
+.wrap{
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis !important;
+    word-break: break-all;
+    word-wrap: break-word;
+    overflow-wrap: initial;
+}
+
+.no-wrap{
+    overflow: hidden;
+    text-overflow: ellipsis !important;
+    overflow-wrap: initial;
+}
+
+.bottom-card:hover{
+    opacity: 80%;
+}
+
+.card:hover{
+    transform: scale(1.05);
+}
+
+.grid-container{
```

nitpicking here, but add a space between each bracket in this file like:
```suggestion
.grid-container {
```